shimming-toolbox/shimmingtoolbox/cli/b0shim.py

```diff
@@ -1196,7 +1127,7 @@ def _save_nii_to_new_dir(list_fname, path_output):
         nib.save(nii, fname_to_save)
 
 
-def _get_fatsat_option(json_anat, fatsat):
+def _get_fatsat_option(nii_anat, fatsat):
```

Comment:
This could be in NiftiFile

Reply:
[6212e7f](https://github.com/shimming-toolbox/shimming-toolbox/pull/614/commits/6212e7fe6aef272bd7e116dcf66cab753d2ee0d6)